server/events/plan_command_runner.go

```diff
@@ -64,6 +67,7 @@ type PlanCommandRunner struct {
 	autoMerger                 *AutoMerger
 	parallelPoolSize           int
 	pullStatusFetcher          PullStatusFetcher
+	locker                     locking.Locker
```

Comment:
This name does not follow the naming conversion of the struct

Reply:
@jamengual I've now fixed this